.script/yamlFileValidator.ts

```diff
@@ -10,6 +10,7 @@ export async function IsValidYamlFile(filePath: string): Promise<ExitCode> {
 }
 
 let fileTypeSuffixes = ["yaml", "yml"];
+let fileKinds = ["Added", "Modified"];
```

Comment:
Nice :)


Please update the readme file with this property:
https://github.com/Azure/Azure-Sentinel/blob/master/.script/ReadMe.md
(in the description about "runCheckOverChangedFiles" function)